pom.xml

```diff
@@ -44,7 +44,7 @@
     </prerequisites>
 
     <properties>
-        <cxf-buildtools-version>3.4.5-SNAPSHOT</cxf-buildtools-version>
+        <cxf-buildtools-version>4.0.0-SNAPSHOT</cxf-buildtools-version>
         <jdk.version>1.8</jdk.version>
```

Comment:
Shouldn't the JDK be bumped as well?

Reply:
@coheigea good one, it should, I could do this in next pull request

Reply:
OK sounds good @reta